src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--1">
+      <div class="stars__star-1"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--2">
+      <div class="stars__star-1"></div>
```

Comment:
![image](https://github.com/mate-academy/layout_stars/assets/80700247/1f5b45a2-1d4b-4c99-adf4-9f513c975c49)

use modifier [nth-child](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child) rule to add active (yellow) stars